src/main/java/io/github/railroad/Railroad.java

```diff
@@ -1,69 +1,69 @@
 package io.github.railroad;
 
-import io.github.railroad.config.Configs;
-import io.github.railroad.drp.DiscordRichPresenceManager;
-import io.github.railroad.objects.ConfirmWindow;
-import io.github.railroad.objects.RailroadTopMenu;
-import io.github.railroad.objects.SelectVersionWindow;
-import io.github.railroad.platform.PlatformType;
-import io.github.railroad.utility.UIUtils;
+import io.github.railroad.menu.TopMenu;
 import javafx.application.Application;
 import javafx.scene.Node;
 import javafx.scene.Scene;
 import javafx.scene.image.Image;
 import javafx.scene.layout.BorderPane;
 import javafx.stage.Stage;
 
+import java.io.IOException;
+import java.io.InputStream;
+
+import static io.github.railroad.lang.LangManger.ENGLISH;
+import static io.github.railroad.lang.LangManger.getLocalization;
+import static io.github.railroad.objects.ConfirmQuitWindow.displayQuitWindow;
+import static io.github.railroad.utility.Components.StageFactory.convertToBuilder;
+
 public class Railroad extends Application {
+    public static boolean darkMode = true;
```

Comment:
Im guessing this is temporary if we want themes.

Reply:
It is, but this is a refactor, and it's not our place to change. We'll come up with a better system in the future.